requirements.txt

```diff
@@ -0,0 +1,7 @@
+torch
```

Comment:
We don't use requirements.txt in OLMo. We use pyproject.toml.

Reply:
Oh woops. I added this for gantry. I'll change locations on this.

Reply:
Removed --- it was created to troubleshoot.